src/api/index.js

```diff
@@ -42,6 +43,7 @@ const api = {
       graphToMarkovChain,
     },
     graph,
+    textEncoding,
```

Comment:
Why expose these directly? Can we remove them?

Reply:
I had exposed them for testing purposes, but they can be removed